src/main/resources/static/views/projects/schedule.html

```diff
@@ -0,0 +1,394 @@
+                                        </div>
+                                    </li>
+                                    <li class="list-group-item">
+                                        12:00 PM to 1:00 PM
+                                        <div class="material-switch pull-right">
+                                            <input id="MondayOption4" name="MondayOption4" type="checkbox"/>
+                                            <label for="MondayOption4" class="label-success"></label>
+                                        </div>
+                                    </li>
+                                    <li class="list-group-item">
+                                        1:00 PM to 2:00 PM
+                                        <div class="material-switch pull-right">
+                                            <input id="MondayOption5" name="MondayOption5" type="checkbox"/>
+                                            <label for="MondayOption5" class="label-success"></label>
+                                        </div>
+                                    </li>
+                                    <li class="list-group-item">
+                                        2:00 PM to 3:00 PM
+                                        <div class="material-switch pull-right">
+                                            <input id="MondayOption6" name="MondayOption6" type="checkbox"/>
+                                            <label for="MondayOption6" class="label-success"></label>
+                                        </div>
+                                    </li>
+                                    <li class="list-group-item">
+                                        3:00 PM to 4:00 PM
+                                        <div class="material-switch pull-right">
+                                            <input id="MondayOption7" name="MondayOption7" type="checkbox"/>
+                                            <label for="MondayOption7" class="label-success"></label>
+                                        </div>
+                                    </li>
+                                    <li class="list-group-item">
+                                        4:00 PM to 5:00 PM
+                                        <div class="material-switch pull-right">
+                                            <input id="MondayOption8" name="MondayOption8" type="checkbox"/>
+                                            <label for="MondayOption8" class="label-success"></label>
+                                        </div>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <br>
+                <button type="button" class="btn btn-secondary" style="float: right;">Submit Availability</button>

```

Comment:
is there a reason why we are using btn-secondary and not btn-suceess as the class?